tools/rgrnastar/macros.xml

```diff
@@ -56,8 +56,8 @@
             <citation type="doi">10.1093/bioinformatics/bts635</citation>
         </citations>
     </xml>
-    <xml name="@SJDBOPTIONS@" token_optional="true">
```

Comment:
At the places where the macro is used you should remove the argument `optional="..."` if its no longer supported. 

There was one use of `<expand macro="@SJDBOPTIONS@" />`, i.e. where optional was `true` before and now it defaults to `false`. Is this desired?

Reply:
Cool. Understood.